README.md

```diff
@@ -122,13 +122,18 @@ Navigate to [https://graph.api.smartthings.com](https://graph.api.smartthings.co
         2. If you **Uninstall** from **AlarmDecoder service** screen it will attempt to automatically remove all sub devices if they are not in use by SHM or other rules.
         3. You can remove blocking child items from the **My Devices** -> **Show Device** screen by selecting the **In Use By** item and deleting it.
 
+### Obtain an API key from the AlarmDecoder webapp
+1. Navigate to the API section of webapp on your local network: ([https://alarmdecoder.local/api/](https://alarmdecoder.local/api/))
+2. Click **Manage API keys**
+3. Click **Generate** for the desired webapp user (eg. `admin`)
+
 ### Configure AlarmDecoder device
 * Using the SmartThings app **on your phone**
     1. Open up the SmartThings app **on your phone**
     2. Tap **My Home** and select the **Things** tab
     3. Select the **AlarmDecoder** device
-    4. Tap the gear icon and select **Edit Device**
```

Comment:
In my SmartThings classic app, I didn't see an **Edit Device** option. Rather, when I clicked the gear, it just went into edit mode. Perhaps they changed this behavior since these instructions were first written.